spec/unit/app/models/edition/workflow_completion_spec.rb

```diff
@@ -55,5 +55,12 @@
         expect(path).to eq("/editions/123/workflow/confirmation")
       end
     end
+
+    describe "when the save_action is 'save_as_draft'" do
+      it "should return the path to redirect to" do
```

Comment:
again, might it be marginally more helpful to be explicit here with something like:

> it returns the path to the document 'show' page